lib/tasks/evm_application.rb

```diff
@@ -102,11 +102,12 @@ def self.output_workers_status(servers)
            w.miq_server_id,
            w.queue_name || w.uri,
            w.started_on && w.started_on.iso8601,
-           w.last_heartbeat && w.last_heartbeat.iso8601]
+           w.last_heartbeat && w.last_heartbeat.iso8601,
+           (w.proportional_set_size || w.memory_usage || 0) / 1.megabyte]
```

Comment:
I know I had this really gross ternary operator for you in a PM as an alternative:

```ruby
(mem = (w.proportional_set_size || w.memory_usage)).nil? ? 0 : mem / 1.megabyte
```

While what you have effectively does the exact same thing without requiring some TUMS™ afterwords, the advantage of the ternary is instead of returning a zero, we could return a `"N/A"`, which probably is more accurate.  But this is being a bit pedantic.